src/js/pages/Theme.js

```diff
@@ -19,7 +19,8 @@ const Theme = ({ theme }) => {
       </div>
 
       <div className="wrapper container-fluid">
-        <TileGroupSet groups={topics}
+        <TileGroupSet
+          groups={topics}
           tileKey={'services'}
           groupTitleSubPath={'topics'}
```

Comment:
The more I see this subpath thing everywhere for topic pages and theme pages, I get bad vibes. Maybe there's a better way to define topic links and theme links... or links in general... in just one place. This type of thing seems like it's a middleware transformation that should be done once wherever the topmost data comes through. Maybe in a clean data step at the page level with all the other clean data steps for RouteData. Or in LanguageWrapper for SiteData things. What do you think?

Reply:
We already have methods that build out service links in cleanData.js (cleanServiceLinks, cleanRelatedServiceLinks) ... maybe we can just add more helper methods?